recipes/omniscidb-cuda/recipe/activate.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+export CONDA_OMNISCIDB_BACKUP_PATH=$PATH
+export PATH=$CONDA_PREFIX/opt/omnisci/bin:$PATH
```

Comment:
Is this needed?

Reply:
`omnisci/bin` contains other programs such as `omnisci_server`, `omnisci_web_server`, `omnisql` that need to be exposed to users. We could create symbolic links to `$CONDA_PREFIX/bin` instead of updating `PATH` if that is preferred.

Reply:
Yes. that is preferred. activation and deactivation scripts are sorted alphabetically, so two packages manipulating PATH might not deactivate correctly.

Reply:
The activate scripts are eliminated and program links are created.